packages/fxa-settings/src/components/Settings/index.tsx

```diff
@@ -20,7 +20,8 @@ import PageCreatePassword from './PageCreatePassword';
 import { MfaGuardPageSecondaryEmailAdd } from './PageSecondaryEmailAdd';
 import { MfaGuardPageSecondaryEmailVerify } from './PageSecondaryEmailVerify';
 import { PageDisplayName } from './PageDisplayName';
-import Page2faSetup from './Page2faSetup';
+import MfaGuardedPage2faSetup from './Page2faSetup';
```

Comment:
Nit, consistent naming is hard. We are already getting a few variations. The initial example I provided was `PageMfaGuard`, then someone introduced `MfaGuardPage` which I guess I like better... Let's go with MfaGuardPage as the naming.